proto/ibc/applications/fee/v1/ack.proto

```diff
@@ -11,4 +11,5 @@ import "gogoproto/gogo.proto";
 message IncentivizedAcknowledgement {
   bytes  result                  = 1;
   string forward_relayer_address = 2 [(gogoproto.moretags) = "yaml:\"forward_relayer_address\""];
+  bool successful = 3;
```

Comment:
@AdityaSripal @colin-axner We can't name this field `Success` as it causes a conflict with the `Success()` function. wdyt?

Reply:
Ahh ok, fine successful will work. Is there some proto option to override this error @colin-axner ?

Reply:
Maybe something more specific? `underlying_app_success`?